rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -500,11 +510,19 @@ void game_list_frame::doubleClickedSlot(const QModelIndex& index)
 	}
 
 	QString category = qstr(m_game_data[i].info.category);
-	
+	std::string game_dir	= m_game_data[i].dir;
+
 	// Boot these categories
 	if (category == category::hdd_Game || category == category::disc_Game || category == category::audio_Video)
 	{
-		const std::string& path = Emu.GetGameDir() + m_game_data[i].info.root;
+		const std::string& path = game_dir + m_game_data[i].info.root;
+
+		if (qstr(path).contains("/dev_hdd0/game/") && fs::is_file(path + "/PS3_GAME/USRDIR/EBOOT.BIN") && fs::is_file(path + "/PS3_DISC.SFB"))
```

Comment:
Some people delete SFB files or have them renamed. Better to make it OR instead of AND